src/main/steps/applicant1Sequence.ts

```diff
@@ -451,12 +454,20 @@ export const applicant1PostSubmissionSequence: Step[] = [
   {
     url: PAYMENT_CALLBACK_URL,
     getNextStep: data =>
-      data.applicationType === ApplicationType.JOINT_APPLICATION ? JOINT_APPLICATION_SUBMITTED : APPLICATION_SUBMITTED,
+      data.applicationType === ApplicationType.JOINT_APPLICATION
+        ? JOINT_APPLICATION_SUBMITTED
+        : data.applicant1SolicitorRepresented === YesOrNo.YES
+          ? APP_REPRESENTED
+          : APPLICATION_SUBMITTED,
```

Comment:
Just checking - to make it consistent with the logic after CHECK_ANSWERS_URL from [line 415 to 421](https://github.com/hmcts/nfdiv-frontend/pull/3727/files#diff-36f0f3fc96f7de9050557fc383bf8334aa0aeb63cfe9b3a5c014a0609a8dc5a3R415) should the condition for applicant1SolicitorRepresented come first?

```suggestion
      data.applicant1SolicitorRepresented === YesOrNo.YES
              ? APP_REPRESENTED
              : data.applicationType === ApplicationType.JOINT_APPLICATION
                  ? JOINT_APPLICATION_SUBMITTED
                   : APPLICATION_SUBMITTED,
```

Reply:
oh, this is a bug, thanks